src/warnet/services.py

```diff
@@ -24,33 +32,28 @@
             "-docker_only=true",
             "-storage_duration=1m0s",
             "-disable_metrics=advtcp,cpu_topology,cpuset,hugetlb,memory_numa,process,referenced_memory,resctrl,sched,tcp,udp",
-        ]
+        ],
     },
     "forkobserver": {
-        "backends": ["compose"],
         "image": "b10c/fork-observer:latest",
         "container_name_suffix": "fork-observer",
         "warnet_port": "23001",
         "container_port": "2323",
         "config_files": [f"{FO_CONF_NAME}:/app/config.toml"],
     },
     "addrmanobserver": {
-        "backends": ["compose"],
         "image": "b10c/addrman-observer:latest",
         "container_name_suffix": "addrman-observer",
         "warnet_port": "23005",
         "container_port": "3882",
         "config_files": [f"{AO_CONF_NAME}:/app/config.toml"],
     },
     "grafana": {
-        "backends": ["compose"],
         "image": "grafana/grafana:latest",
```

Comment:
I worry it's confusing to leave in services that were compose only and remove the key that marks them as compose only. Grafana is configured separately in k8s using the `install_logging.sh` script for example.

I think it gives the impression that these things are available to be added to your graph file and have them run.

Reply:
OK removed a few services (unlikelty to be re-added), but commented out a few which IMO we want back (mainly the *observer services).